src/components/loader-bar/loader-bar.stories.mdx

```diff
@@ -1,5 +1,7 @@
 import { useState } from "react";
 import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import LoaderBar from ".";
+import Button from "../button";
```

Comment:
```suggestion
```
You have a repeated import of `LoaderBar` and `Button` is not used in this file, so both can be removed here.